bootloader/prebootloader/src/rtl872x/part1/include.mk

```diff
@@ -1,8 +1,20 @@
 MAIN_STACK_SIZE = 4096
 
+# # FIXME
```

Comment:
Are these fixed in the prebootloader-part1 makefile below?

Reply:
Yes, this should probably be removed.